prepare_cyrus.hs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env stack
+-- stack runghc --resolver lts-8.14 --package turtle --install-ghc
```

Comment:
I'm adding my 2 cents to this : if you don't do bash (and I can very well understand why you would not) you could do Scala (you already saw my scala script BTW) so we would have a single language for gatling + automation